### DISCUSSION TITLE
Unable to configure plugin

### DISCUSSION THREAD

**Discussion by watsonix** [ObsidianPublisher/obsidian-github-publisher#246] [Q&A — Plugin]
I must be missing something . The documentation states "Configure the plugin’s options by setting the repository name, GitHub username, GitHub token, and branch name." but I see nowhere to do this. Unlike other plugins I see no "Options" button once I enable it.

**Comment by Mara-Li** [comment]
Seems there are a problem when you enable it. Please open a new issue.
